app/views/responsible_body/mobile/extra_data_requests/index.html.erb

```diff
@@ -10,23 +10,20 @@
   </ol>
   </div>
 <% end %>
-
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
     <h1 class="govuk-heading-xl">
       <%= t('page_titles.request_extra_mobile_data') %>
     </h1>
-
     <% unless @extra_mobile_data_requests.any? %>
       <%= render partial: 'shared/use_the_mobile_guide' %>
     <% end %>
-
-    <p class="govuk-body">
-      <%= govuk_button_link_to 'Request data for someone', new_responsible_body_extra_mobile_data_request_path %>
-    </p>
+    <%= govuk_button_link_to 'New request',
+      responsible_body_mobile_extra_data_requests_type_path,
+      class: 'govuk-button govuk-!-margin-top-5'
```

Comment:
I think the spacing here is wrong too. I don't think we need the class attr here. But should still be wrapped in a paragraph.

Reply:
Done